GoogleUtilities/Environment/Public/GoogleUtilities/GULSecureCoding.h

```diff
@@ -25,7 +25,7 @@ NS_ASSUME_NONNULL_BEGIN
                                 fromData:(NSData *)data
                                    error:(NSError **)outError;
 
-+ (nullable id)unarchivedObjectOfClass:(Class)class
++ (nullable id)unarchivedObjectOfClass:(Class)aClass
```

Comment:
Would this change need to wait for a major version bump?  At first, I thought yes, but am now less sure since the method signature/type remains the same.

Reply:
My hunch is that this would be fine in a patch release since even in less common scenarios, like calling indirectly with ObjC runtime methods, the param name isn't involved (at least not in the ones I've seen). Might be good to wait for Paul to give a second opinion though.

Reply:
Agreed, it shouldn't be breaking - and as far as I can tell, this API isn't being used and may not have been for awhile.

Reply:
Thank you both!